src/react-components/ui-root.js

```diff
@@ -595,6 +595,9 @@ class UIRoot extends Component {
       if (micDeviceId) {
         this.props.store.update({ settings: { lastUsedMicDeviceId: micDeviceId } });
       }
+      const mediaStreamForMicAnalysis = new MediaStream();
+      mediaStreamForMicAnalysis.addTrack(this.state.audioTrack.clone());
+      this.props.scene.emit("local-media-stream-created", { mediaStream: mediaStreamForMicAnalysis });
```

Comment:
I didn't think of a good way to "future proof" accidental mis-use of this mediaStream. When toggling your microphone, the janus adapter makes sure to disable the audio streams it knows about, so that's good. In general no one should be piping audio from this mediaStream into anything that is sent over the networked or saved to a file (like videos from the camera tool).